lib/routes/sohumobile/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import type { Namespace } from '@/types';
```

Comment:
The site does not have `sohumobile.com` in its second level domain. Do not [create a namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) that does not follow its second level domain.